src/wallet/tx_builder.rs

```diff
@@ -147,6 +147,7 @@ pub(crate) struct TxParams {
     pub(crate) add_global_xpubs: bool,
     pub(crate) include_output_redeem_witness_script: bool,
     pub(crate) bumping_fee: Option<PreviousFee>,
+    pub(crate) current_height: Option<u32>,
```

Comment:
I feel it would be really helpful for lib users if these fields can come with their comments explaining what are they for.. 

Doesn't have to be included in this PR. But something for future..

Reply:
When it's going to be exposed publicly, sure :) For now just the devs are seeing it, and I'd say it's easy enough to understand the various params, since the functions modifying them are commented anyways...

Reply:
It has a TODO to be exposed publicly, and as internal dev usage also I feel its better to document them to disclose their intents.. 

But yes not blocking on this PR. Can be done later..